File: scripts/read_cnpj.py
import PyPDF2

try:
    with open('CNPJ SOLAR AMÉRICA ATUALIZADO.pdf', 'rb') as file:
        reader = PyPDF2.PdfReader(file)
        text = ''
        for page in reader.pages:
            text += page.extract_text() + '\n'
        print(text)
except Exception as e:
    print(f"Error: {e}")

File: scripts/read_cnpj.mjs
import fs from 'fs';
import pdf from 'pdf-parse/lib/pdf-parse.js';

const dataBuffer = fs.readFileSync('CNPJ SOLAR AMÉRICA ATUALIZADO.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(err => {
    console.error("Error reading PDF:", err);
});
